src/kits/tracker/ContainerWindow.cpp

```diff
@@ -1010,13 +968,41 @@ BContainerWindow::Init(const BMessage* message)
 	SetFlags(Flags() & ~B_NO_WORKSPACE_ACTIVATION);
 }
 
+void
+BContainerWindow::InitLayouts()
+{
+	BLayoutItem* item = 
+		fPoseViewHolder->GroupLayout()->AddView(0, fPoseView->TitleView());
+	BSize minSize = item->MinSize();
+	BSize maxSize = item->MaxSize();
+	item->SetExplicitMinSize(BSize(minSize.Width(), kTitleViewHeight));
+	item->SetExplicitMaxSize(BSize(maxSize.Width(), kTitleViewHeight));
+		
+	item = fCountContainer->GroupLayout()->AddView(fPoseView->CountView());
+	minSize = item->MinSize();
+	maxSize = item->MaxSize();
+	item->SetExplicitMinSize(BSize(kCountViewWidth, minSize.Height()));
+	item->SetExplicitMaxSize(BSize(kCountViewWidth, maxSize.Height()));
+
+	// Create a container for each scroll bar
+	// so we can eliminate the extra border
```

Comment:
I thought about this, but I couldn't find anything in the scrollbar code that allowed changing the borders...